index.html

```diff
@@ -98,6 +98,7 @@
         <option value="progress-upload-real">Real Upload Events</option>
         <option value="response-type">Response Type</option>
         <option value="events">Events</option>
+        <option value="event-target">Event Target</option>
```

Comment:
This is unnecessary as it is not a use case.

Reply:
`event-target.html` removed